Emby.Server.Implementations/Services/SwaggerService.cs

```diff
@@ -0,0 +1,259 @@
+            {
+                host = uri.Host;
+            }
+
+            var securitySchemes = new Dictionary<string, SwaggerSecurityScheme>();
+
+            securitySchemes["api_key"] = new SwaggerSecurityScheme
+            {
+                name = "api_key",
+                type = "apiKey",
+                @in = "query"
+            };
+
+            var spec = new SwaggerSpec
+            {
+                schemes = new[] { "http" },
+                tags = GetTags(),
+                swagger = "2.0",
+                info = new SwaggerInfo
+                {
+                    title = "Jellyfin Server API",
+                    version = "1.0.0",
+                    description = "Explore the Jellyfin Server API",
+                    contact = new SwaggerConcactInfo
+                    {
+                        name = "Jellyfin Community",
+                        url = "https://jellyfin.readthedocs.io/en/latest/user-docs/getting-help/"
+                    }
+                },
+                paths = GetPaths(),
+                definitions = GetDefinitions(),
+                basePath = "/jellyfin",
+                host = host,
+
+                components = new SwaggerComponents
+                {
+                    securitySchemes = securitySchemes
+                }
+            };
+
+            return spec;
+        }
+
+
+        private SwaggerTag[] GetTags()
+        {
+            return new SwaggerTag[] { };
+        }
+
+        private Dictionary<string, SwaggerDefinition> GetDefinitions()
+        {
+            return new Dictionary<string, SwaggerDefinition>();
+        }
+
+        private IDictionary<string, Dictionary<string, SwaggerMethod>> GetPaths()
+        {
+            var paths = new SortedDictionary<string, Dictionary<string, SwaggerMethod>>();
+
+            var all = ServiceController.Instance.RestPathMap.OrderBy(i => i.Key, StringComparer.OrdinalIgnoreCase).ToList();
+
+            foreach (var current in all)
+            {
+                foreach (var info in current.Value)
+                {
+                    if (info.IsHidden)
+                    {
+                        continue;
+                    }
+
+                    if (info.Path.StartsWith("/mediabrowser", StringComparison.OrdinalIgnoreCase))
+                    {
+                        continue;
+                    }
+                    if (info.Path.StartsWith("/jellyfin", StringComparison.OrdinalIgnoreCase))
+                    {
+                        continue;
+                    }
+
+                    paths[info.Path] = GetPathInfo(info);
+                }
+            }
+
+            return paths;
+        }
+
+        private Dictionary<string, SwaggerMethod> GetPathInfo(RestPath info)
+        {
+            var result = new Dictionary<string, SwaggerMethod>();
+
+            foreach (var verb in info.Verbs)
+            {
+                var responses = new Dictionary<string, SwaggerResponse>
```

Comment:
```suggestion
                var responses = new Dictionary<string, SwaggerResponse>();
```
No need to take up 3 lines.

Reply:
@Bond-009 I'll be rewriting this to support OpenAPI spec 3.0. Style changes will be addressed then.